src/ui/components/shared/Nags/Nags.tsx

```diff
@@ -59,21 +57,3 @@ export function EditorNag() {
 
   return <NagHat subText="Now hover on a line number" nagType={Nag.FIRST_BREAKPOINT_ADD} />;
 }
-
-export function ConsoleNag() {
-  const { messageIDs } = useAppSelector(getVisibleMessageData);
-
-  // Don't show the console nag that directs the user to click on one of the console messages
-  // if there aren't any console messages to begin with.
-  if (!messageIDs.length) {
-    return null;
-  }
-
-  return (
-    <NagHat
-      mainText="Want to see something cool?"
```

Comment:
:question: Do we still have this in the new code as part of the onboarding flow?

Reply:
Unfortunately we probably do! Will follow up via FE-713